changedetectionio/tests/unit/test_notification_diff.py

```diff
@@ -26,6 +26,12 @@ def test_expected_diff_output(self):
         self.assertIn('(removed) for having learned computerese,', output)
         self.assertIn('(removed) I continue to examine bits, bytes and words', output)
 
+        # UTF-8 test, show failure
+        output = diff.render_diff(previous_file=base_dir + "/test-content/before.txt",
+                                  newest_file=base_dir + "/test-content/after-3.txt")
+        output = output.split("\n")
+        print(output)
```

Comment:
tests should never prove something by printing, can you please change this to "assert" that the string shows and that the broken string doesnt show?

Reply:
changed. 
the original code couldn't reach the print line, because failed in render_diff function